tests/core/baseline_test.py

```diff
@@ -58,6 +58,30 @@ def test_basic_usage(self, path):
         assert len(results['test_data/files/file_with_secrets.py']) == 1
         assert len(results['test_data/files/tmp/file_with_secrets.py']) == 2
 
+    def test_with_multiple_files(self):
+        results = self.get_results(path=[
+            'test_data/files/file_with_secrets.py',
+            'test_data/files/tmp/file_with_secrets.py',
+        ])
+
+        assert len(results['test_data/files/file_with_secrets.py']) == 1
+        assert len(results['test_data/files/tmp/file_with_secrets.py']) == 2
+        assert 'test_data/files/file_with_secrets.py' in results
+        assert 'test_data/files/tmp/file_with_secrets.py' in results
+
+    def test_with_multiple_non_existent_files(self):
+        results = self.get_results(path=['non-existent-file.A', 'non-existent-file.B'])
+
+        # No expected results, because files don't exist
+        assert not results
+
+    def test_with_folders_and_files(self):
+        results = self.get_results(path=['test_data/', 'non-existent-file.B'])
+
+        assert len(results.keys()) == 10
```

Comment:
Is there a better way you can do this?

My only concern for this magic number is:

1. Another developer creates a new test file under `test_data/`
2. They write a new test, that leverages that new file. This works.
3. For some bizarre reason, this test fails now.

The CI will catch failing tests, but it's just less convenient if there's a better option.

Reply:
Added a couple of known asserts (to check that directory scan works as expected) and removed the hardcoded test!